apps/condo/domains/miniapp/schema/B2BAppAccessRight.js

```diff
@@ -32,8 +61,28 @@ const B2BAppAccessRight = new GQLListSchema('B2BAppAccessRight', {
         delete: false,
         auth: true,
     },
+    hooks: {
+        validateInput: async ({ resolvedData, context, existingItem }) => {
+            const newItem = { ...existingItem, ...resolvedData }
+            const accessRightSetId = get(newItem, 'accessRightSet')
+
+            if (accessRightSetId) {
+                const appId = get(newItem, 'app')
+                const accessRightSet = await getByCondition('B2BAppAccessRightSet', {
+                    app: { id: appId, deletedAt: null },
+                    deletedAt: null,
+                    id: accessRightSetId,
+                })
+
+                if (!accessRightSet) {
+                    throw new GQLError(ERRORS.ACCESS_RIGHT_SET_NOT_FOUND, context)
```

Comment:
It's not exactly "Not found" for user. It does not match a criteria (to have same app as accessRight)